tests.py

```diff
@@ -840,6 +847,70 @@ def create_test_pull_job(namespace, quay_host, username, password, concurrency,
     logger.info("Created Job: %s", resp.metadata.name)
 
 
+def parallel_process(user, **kwargs):
+    common_args = kwargs
+    # Container Operations
+    redis_client.delete('tags_to_push'+"-".join(user.split("_")))  # avoid stale data
```

Comment:
are the parallel operations only for push/pull?

Reply:
No. For rest of the APIS, we hit endpoints using `vegeta`. So the concurrency is automatically handled by the '--rate' option in the tool. Since we execute podman commands for push/pull we have to explicitly implement parallel processing for them as in the above code.